pages/previewModel.js

```diff
@@ -120,13 +124,14 @@ function updatePreview() {
     // give them all the same special ID so that we can easily remove them later.
     // We then do something similar for link (CSS) elements.
     clearRemovableHeadElements();
-    var scriptElements = content.getElementsByTagName('script')
-    for (var i = 0; i < scriptElements.length; i++) {
+    var i;
```

Comment:
For my own knowledge, is there really a performance gain by moving `i` outside the scope of the for loops, just because we have multiple for loops?  I've generally viewed it as dangerous ... the logic being that we should keep variables declared at the smallest possible scope that makes sense.  For a simple iterator, it seems that defining it directly within the for loop is the best way to go.  Again -- I'm a bit less familiar with JavaScript, so maybe I'm missing something here.  Certainly within a C/C++/C# app, I would keep `i` declared within the context of the for loop.